src/python_testing/TC_RR_1_1.py

```diff
@@ -0,0 +1,432 @@
+
+        asserts.assert_equal(len(client_list), num_fabrics_to_commission *
+                             num_controllers_per_fabric, "Must have the right number of clients")
+
+        client_by_name = {client.name: client for client in client_list}
+
+        # Step 2: Set the Label field for each fabric and BasicInformation.NodeLabel to 32 characters
+        logging.info("Step 2: Setting the Label field for each fabric and BasicInformation.NodeLabel to 32 characters")
+
+        for idx in range(num_fabrics_to_commission):
+            fabric_number = idx + 1
+            # Client is client A for each fabric to set the Label field
+            client_name = "RD%dA" % fabric_number
+            client = client_by_name[client_name]
+
+            # Send the UpdateLabel command
+            label = ("%d" % fabric_number) * 32
+            logging.info("Step 2a: Setting fabric label on fabric %d to '%s' using client %s" % (fabric_number, label, client_name))
+            await client.SendCommand(self.dut_node_id, 0, Clusters.OperationalCredentials.Commands.UpdateFabricLabel(label))
+
+            # Read back
+            fabric_metadata = await self.read_single_attribute(client, node_id=self.dut_node_id, endpoint=0, attribute=Clusters.OperationalCredentials.Attributes.Fabrics)
+            print(fabric_metadata)
+            asserts.assert_equal(fabric_metadata[0].label, label, "Fabrics[x].label must match what was written")
+
+        # Before subscribing, set the NodeLabel to "Before Subscriptions"
+        logging.info("Step 2b: Set BasicInformation.NodeLabel to '%s'" % BEFORE_LABEL)
+        await client_list[0].WriteAttribute(self.dut_node_id, [(0, Clusters.Basic.Attributes.NodeLabel(value=BEFORE_LABEL))])
+
+        node_label = await self.read_single_attribute(client, node_id=self.dut_node_id, endpoint=0, attribute=Clusters.Basic.Attributes.NodeLabel)
+        asserts.assert_equal(node_label, BEFORE_LABEL, "NodeLabel must match what was written")
+
+        # Step 3: Add 3 Access Control entries on DUT with a list of 4 Subjects and 3 Targets with the following parameters (...)
+        logging.info("Step 3: Fill ACL table so that all minimas are reached")
+
+        for idx in range(num_fabrics_to_commission):
+            fabric_number = idx + 1
+            # Client is client A for each fabric
+            client_name = "RD%dA" % fabric_number
+            client = client_by_name[client_name]
+
+            acl = self.build_acl(fabric_number, client_by_name, num_controllers_per_fabric)
+
+            logging.info("Step 3a: Writing ACL entry for fabric %d" % fabric_number)
+            await client.WriteAttribute(self.dut_node_id, [(0, Clusters.AccessControl.Attributes.Acl(acl))])
+
+            logging.info("Step 3b: Validating ACL entry for fabric %d" % fabric_number)
+            acl_readback = await self.read_single_attribute(client, node_id=self.dut_node_id, endpoint=0, attribute=Clusters.AccessControl.Attributes.Acl)
```

Comment:
Have you tried doing a fabric-scoped read? That might obviate the need to do the patch-up below.

Not urgent/critical, can follow-up.